README.md

```diff
@@ -14,7 +14,7 @@ Add https://github.com/Bandwidth/webrtc-swift.git as a dependency to your Packag
 
 ```swift
 dependencies: [
-    .package(url: "https://github.com/Bandwidth/webrtc-swift.git", .upToNextMajor(from: "1.0.0"))
+    .package(name: "webrtc", url: "https://github.com/Bandwidth/webrtc-swift.git", .upToNextMajor(from: "1.0.0"))
```

Comment:
should it be `bandwidth-webrtc` to avoid confusion with any public webrtc packages?

Reply:
We are _the_ WebRTC solution for including WebRTC into your Swift projects. A more serious and complete answer is that this repo contains the built WebRTC libraries themselves and is "WebRTC" for Apple platforms. Our other repo is named similarly to what you mention though which contains the Bandwidth additions.